core/domain/skill_domain.py

```diff
@@ -585,6 +585,16 @@ def require_valid_skill_id(cls, skill_id):
         if len(skill_id) != 12:
             raise utils.ValidationError('Invalid skill id.')
 
+    @classmethod
+    def require_valid_skill_ids(cls, skill_ids):
```

Comment:
@Showtim3, sorry for the misunderstanding, I thought of creating a `_validate_skill_ids ` method in the controller files itself. Let's not add this to the domain layer as we expect these class to handle one object instead of list of items.

This will need a re-work (sorry about that), Let me know if I can help anyway!

Reply:
Hey @DubeySandeep No problem at all, though there's a doubt that i have, Should that function ```validate_skill_ids``` should be in the controller or the service layer?

Reply:
controller layer* (As it's kinda controller specific*)

Reply:
We have this pattern in other controllers as well you can follow that (like suggestion.py)!

Reply:
Done.